.github/workflows/ci.yaml

```
@@ -6,61 +6,52 @@ on:
       - 'jenkins-scripts/dsl/**'
```

```suggestion
      - 'jenkins-scripts/dsl/**'
```

Does it make sense to update the triggering mechanisms here so that updates only occur on the main branch or make any other affordances to prevent abuse of these workflows?

I think the risk is limited to the logs directory without very obvious interference with the workflows and since this is a metadata update not a production deployment we don't need to lock this down as tightly. It is also nice to have the change auto-committed _before_ the PR is merged rather than after, which is what my alternative strategy requires.

---

> This will cover direct commits to `master` which would be a good thing to have. However, as you mentioned, showing the diff on the PR before it is merged is really useful.

Yeah this was the alternative I was trying to describe. I think that for this type of change, having it performed on the PR branch before merge is preferred. So it's good as is.